x/mongo/driver/topology/server.go

```diff
@@ -97,6 +98,7 @@ type Server struct {
 	// description related fields
 	desc                   atomic.Value // holds a description.Server
 	updateTopologyCallback atomic.Value
+	topologyID             atomic.Value // holds a primitive.ObjectID
```

Comment:
Why does this need to be an `atomic.Value` rather than a `primitive.ObjectID`?

Reply:
done